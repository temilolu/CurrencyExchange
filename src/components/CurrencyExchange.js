import React from 'react';

const CurrencyExchange = ({ base, convertTo, handleInput, disable }) => {
	return (
		<React.Fragment>
			<form>
				<div className='input-group border rounded p-3'>
					<input
						type='number'
						disabled={disable}
						className='form-control-plaintext'
						value={base}
						onChange={handleInput}
					/>

					<div className='input-group-addon my-3 font-weight-light fsizes'>
						{convertTo}
					</div>
				</div>
			</form>
		</React.Fragment>
	);
};

export default CurrencyExchange;
