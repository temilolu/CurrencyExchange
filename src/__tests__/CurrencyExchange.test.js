import React from 'react';
import CurrencyExchange from '../components/CurrencyExchange';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

afterEach(cleanup);

describe('<CurrencyExchange /> spec', () => {
	it('Should matches snapshot', () => {
		const { asFragment } = render(<CurrencyExchange />);
		expect(asFragment()).toMatchSnapshot();
	});
});
