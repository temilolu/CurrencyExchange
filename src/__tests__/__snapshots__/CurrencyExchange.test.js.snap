// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyExchange /> spec Should matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <div
      class="input-group border rounded p-3"
    >
      <input
        class="form-control-plaintext"
        type="number"
        value=""
      />
      <div
        class="input-group-addon my-3 font-weight-light fsizes"
      />
    </div>
  </form>
</DocumentFragment>
`;
